teradatalabs/cdh5-hive-kerberized/files/conf/hive-site.xml

```diff
@@ -99,4 +99,14 @@
         <value>hdfs,hive</value>
     </property>
 
+    <property>
```

Comment:
There were other changes to hive-site.xml that were on the docs you linked to, why are those not necessary?